library/js/oeUI/universalTooltip.js

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Universal jquery tooltip
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author Ranganath Pathak <pathak@scrs1.org>
+ * @copyright Copyright (c) 2018 Ranganath Pathak <pathak@scrs1.org>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+?>
```

Comment:
The universalTooltip.js needs to be included whenever the class OemrUI  is called. Can you think of a convenient place where it can be placed without having to remember yet another script to include

Reply:
Would it make sense to just always include it? (ie. no matter what like the default scripts are in the Header function call)

Reply:
To function it need jquery-ui, not always included in pages. Unless you want to modify the `openemr/config/config.yaml` file

Reply:
Per the class, it looks like both jquery-ui and jquery-ui-base are also required. Is this true? Then will think about best way to easily support this in these scripts.

Reply:
yes the base is also needed. I tried to find a place to put it but could not think of a suitable one, hence the ugly hack